controls/radpanelbar/features/resizeable_items.md

```diff
@@ -44,6 +44,15 @@ The expanded length of the __RadPanelBarItem__ element can be controlled via its
 
 ![expanded item length constraints](../images/panelbaritem-expanded.gif)
 
+## Enabling RadPanelBarItem Scrolling
+
+The __RadPanelBar__ control exposes an additional __IsScrollViewerInsideItemsEnabled__ property that allows to scroll the content of each __RadPanelBarItem__ element. The property controls the __ScrollBarVisibility__ of the __ScrollViewer__ control, which is present inside the __ControlTemplate__ of the __RadPanelBarItem__ control. To allow the __RadPanelBarItem__ control to scroll its content, set the __IsScrollViewerInsideItemsEnabled__ property to __True__.
```

Comment:
"exposes and additional ... property...."

additional to what? I think you can omit the word "additional".

Also, you can separate the last sentence in a new paragraph. The one that says to set the property to True.